src/main/java/generationExamples/ExitBeforeStartedExample.java

```diff
@@ -0,0 +1,32 @@
+package generationExamples;
+
+import main.Configurations;
+import main.Main;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+public class ExitBeforeStartedExample implements EquationPattern{
+    private final static String MESSAGE = "Loser";
```

Comment:
Again it's better to use `EXIT_BEFORE_STARTED_MESSAGE` name.

Reply:
fixed